apps/system/test/unit/notification_screen_test.js

```diff
@@ -31,7 +33,7 @@ var mocksForNotificationScreen = new MocksHelper([
   'SettingsListener',
   'SettingsURL',
   'Service',
-  'LazyLoader'
+  'LazyLoader',
```

Comment:
nit: let's remove this comma, or even better, use the the MocksHelper for loading the new mock